shared/checks/oval/package_uuidd_installed.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+  <definition class="compliance" id="package_uuidd_installed"
```

Comment:
To automatically generate remediations and OVAL, add ``uuidd`` to the package template file here:
https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/templates/csv/packages_installed.csv

Can then remove the entry from ol7/templates/ and ``git rm`` this file.